apps/backoffice-v2/src/domains/customer/fetchers.ts

```diff
@@ -18,6 +18,16 @@ export const fetchCustomer = async () => {
         customerStatus: z.string().optional(),
         country: z.union([z.string(), z.null()]).optional(),
         language: z.union([z.string(), z.null()]).optional(),
+        config: z
+          .object({
+            isMerchantMonitoringEnabled: z.boolean().default(false),
+            isExample: z.boolean().default(false),
+          })
+          .nullable()
+          .default({
```

Comment:
is the `default({...})` needed when we have defaults on the inner properties?

Reply:
Yes. You are supplying a default for when the root property is null or undefined.